wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -112,14 +128,20 @@ class Robot : public frc::TimedRobot {
   void TeleopPeriodic() override {
     if (m_joystick.GetTrigger()) {
       // Here, we run PID control like normal, with a constant setpoint of 30in.
-      double pidOutput = m_controller.Calculate(m_encoder.GetDistance(),
-                                                units::meter_t{30_in}.value());
-      m_motor.SetVoltage(units::volt_t{pidOutput});
+      m_controller.SetGoal(units::meter_t{30_in});
```

Comment:
Perhaps extract the goal to a constant? Besides, I think it doesn't need the explicit conversion to meters.

Reply:
Correct, it autoconverts implicitly.